bumpversion/cli.py

```diff
@@ -11,19 +8,18 @@
 import sre_constants
 import sys
 import warnings
+from configparser import (
```

Comment:
let's put imports in alphabetical order

Reply:
Done! https://github.com/c4urself/bump2version/pull/108/commits/1eb846bb7b6c74955262840273b93b744c21341b